src/index.html

```diff
@@ -1,17 +1,210 @@
+            type="date"
+            name="date"
+            id="date"
+          >
+        </div>
+
+        <div class="contacts__form-item">
+          <label for="accept">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            name="accept"
+            id="accept"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="contacts__form-field">
+        <legend>Registration</legend>
+        <div class="contacts__form-item">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            required
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="contacts__form-item">
+          <label for="password">Password:</label>
+          <input
+            type="password"
+            name="password"
+            id="password"
+            minlength="8"
+            maxlength="12"
+            autocomplete="off"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="contacts__form-field">
+
+        <legend>An interesting fact about you!</legend>
+
+        <div class="contacts__form-item">Do you love cats?
+
+          <label>
+            <input
+              type="radio"
+              name="choise"
+              value="yes"
+            >
+            Yes
+          </label>
+
+          <label>
+            <input
+              type="radio"
+              name="choise"
+              value="no"
+            >
+            No
+          </label>
+        </div>
+        <div class="contacts__form-item">
+          <label>What is your favorit color?
+            <input
+              type="color"
+              name="color"
+            >
+
+          </label>
+        </div>
+
+        <div class="contacts__form-item">
+          <label>What time do you go to bed?
+            <input
+              type="time"
+              name="time"
+            >
+          </label>
+        </div>
+
+        <div class="contacts__form-item">
+          What are your favorite brand of cars?
+          <select
+            name="brans"
+            multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="AUDI">Audi</option>
+            <option value="LADA">Lada</option>
+          </select>
+        </div>
+
+        <div class="contacts__form-item">
+          How do you rate our work?
+          <input
+            type="range"
+            name="range"
+            value="0"
+          >
+        </div>
+
+      </fieldset>
+
+      <fieldset class="contacts__form-field">
+        <legend>Additional info</legend>
+        <div class="contacts__form-item">
+          <label for="textarea">
+            Comments:
+          </label>
+          <textarea
+            name="textarea"
+            rows="2"
+            id="textarea"
+          >
+          </textarea>
```

Comment:
```suggestion
          ></textarea>
```
<img width="301" alt="image" src="https://user-images.githubusercontent.com/53611910/202918905-ebd02007-8394-4f35-94c3-476552e4cb56.png">

Reply:
to fix this

Reply:
> 

error  Tag’s closing bracket should be on  the same level as opening one   (closing-and-opening-on-seme-level)

Reply:
html lint error

Reply:
Yes, but it is just a linter cofig. If it is important for the element's behavior, you can skip it